sdk/src/main/java/com/oursky/authgear/latte/Latte.kt

```diff
@@ -461,6 +461,33 @@ class Latte(
         }
     }
 
+    suspend fun getProofOfPhoneNumberVerification(): String {
+        return withContext(Dispatchers.IO) {
+            val url = URL(this@Latte.middlewareEndpoint + "/proof_of_phone_number_verification")
+            val request = ProofOfPhoneNumberVerificationRequest(
+                accessToken = authgear.accessToken!!,
+            )
```

Comment:
We should return a proper error if accessToken is null.
We can also call "refreshAccessTokenIfNeeded" first to ensure the accessToken is valid.
Raise a `UnauthenticatedUserException` if accessToken is null.